config/locales/server.en.yml

```diff
@@ -114,3 +114,22 @@ en:
   reviewable_score_types:
     needs_review:
       title: "Needs Review"
+
+  user_notifications:
+    chat_summary:
+      deleted_user: "Deleted user"
+      description:
+        one: "You have a new chat message"
+        other: "You have new chat messages"
+      from: "%{site_name}"
+      subject:
+        one: "[%{email_prefix}] New chat message"
+        other: "[%{email_prefix}] New chat messages"
```

Comment:
```suggestion
        one: "%{email_prefix} new chat message"
        other: "%{email_prefix} new chat messages"
```

Reply:
Wouldn't the subject look weird with this change? e.g. `Discourse Meta new chat messages` instead of `[Discourse Meta] New chat messages`